package.json

```diff
@@ -16,7 +22,8 @@
     "grunt-contrib-less": "^1.2.0",
     "grunt-contrib-watch": "^0.6.1",
     "grunt-wiredep": "~2.0.0",
-    "less-plugin-autoprefix": "^1.5.1"
+    "less-plugin-autoprefix": "^1.5.1",
+    "patternfly": ">=3.5.1 <4.0"
```

Comment:
When did we end up with a dependency on patternfly in this project?

Reply:
Looks like it's a dev dependency only, so not required. Is it just for testing with patternfly styles?

Reply:
I copied this from the bower.json dev dependencies